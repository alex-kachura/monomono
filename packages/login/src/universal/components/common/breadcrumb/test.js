import React from 'react';
import { Breadcrumb } from './';

describe('[Component: Breadcrumb]', () => {
  let component;

  const mockProps = {
    links: [{ href: 'www.testco.com', text: 'Test page' }, { text: 'Test page2' }],
    config: {
      GB: {
        externalApps: {
          tescoHomepage: 'https://www.testco.com',
        },
      },
    },
    region: 'GB',
    getLocalePhrase: (key) => key,
  };

  beforeEach(() => {
    component = global.contextualShallow(<Breadcrumb {...mockProps} />);
  });

  it('should render correctly', () => {
    expect(component).toMatchSnapshot();
  });
});
