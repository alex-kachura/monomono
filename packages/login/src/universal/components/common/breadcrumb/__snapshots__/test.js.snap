// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`[Component: Breadcrumb] should render correctly 1`] = `
<Styled(base-components__RootElement)>
  <WithTheme(Breadcrumb)
    altLink={[Function]}
    home={
      Object {
        "href": "https://www.testco.com",
      }
    }
    links={
      Array [
        Object {
          "href": "www.testco.com",
          "text": "Test page",
        },
        Object {
          "text": "Test page2",
        },
      ]
    }
  />
</Styled(base-components__RootElement)>
`;
